phoebe/frontend/bundle.py

```diff
@@ -916,7 +916,7 @@ def existing_value(param):
             # call set_hierarchy to force mass constraints to be rebuilt
             b.set_hierarchy()
 
-        if phoebe_version_import < StrictVersion("2.4.0") or ".dev" in version:
+        if phoebe_version_import < StrictVersion("2.4.0"):
             warning = "importing from an older version ({}) of PHOEBE to PHOEBE 2.4.  This may take some time.  Please check all values.".format(phoebe_version_import)
```

Comment:
that's a good point - the 2.4 was intentional (since it is updating to the requirements introduced in 2.4), but I just went through and changed these all to be consistent and say "to PHOEBE 2.x+" instead.